src/include/gurt/common.h

```diff
@@ -586,6 +586,9 @@ int
 d_getenv_bool(const char *name, bool *bool_val);
 int
 d_getenv_char(const char *name, char *char_val);
+/* DAOS-14981 XXX d_getenv_int() is deprecated, please use d_getenv_uint() */
```

Comment:
There's a compiler intrinsic you should use here instead which will make the compiler warn if the function is used.